src/admin/menu/views/MenuDetail.scss

```diff
@@ -29,3 +35,26 @@
 .c-menu-detail__nav-label {
 	margin: 0 auto 0 ($g-spacer-unit * 3);
 }
+
+.c-admin__header {
+	border-bottom: 0.1rem solid #d6dee3;
+	background-color: #edeff2;
```

Comment:
Search for the hard coded colors in the colors.scss file and replace them with variable names. Example:
```
	@import '../../../../styles/settings/colors';

	...

	background-color: $color-gray-50;
```